pkg/apis/kudo/v1beta1/instance_types.go

```diff
@@ -72,7 +72,7 @@ type PlanStatus struct {
 	Status  ExecutionStatus `json:"status,omitempty"`
 	Message string          `json:"message,omitempty"` // more verbose explanation of the status, e.g. a detailed error message
 	// +nullable
-	LastFinishedRun metav1.Time           `json:"lastFinishedRun,omitempty"`
+	LastFinishedRun *metav1.Time          `json:"lastFinishedRun,omitempty"`
```

Comment:
As this can now be `nil`, we should have checks for `nil` in places where this field is used, right?

I.e. in instance_types_helpers.go `wasRunAfter` and plan_history.go `planHistory`

Reply:
yep other than this it lgtm